consensus/hotstuff/signature/packer.go

```diff
@@ -229,3 +168,91 @@ func deserializeFromBitVector(serialized []byte, count int) ([]hotstuff.SigType,
 
 	return types, nil
 }
+
+// encodeSignerIndicesAndSigType encodes the given stakingSigners and beaconSigners into signer indices and sig types.
+// The given fullMembers provides the canonical order of signer index for each signer.
+// For instance, assuming fullMembers are [A,B,C,D,E,F], stakingSigners are [D,E], beaconSigners are [A,B], then
+// the signerIndices will be [byte([1,1,0,1,1,0,0,0])],
+// 			bit 1 indicates the node at that index signed, bit 0 indicates the node at that index didn't sign
+// the sigType will be [byte([1,1,0,0,0,0,0,0])],
+// 			bit 1 indicates the signer at the same index in signerIndices signed random beacon sig
+// 			bit 0 indicates the signer at the same index in signerIndices signed staking sig
+func encodeSignerIndicesAndSigType(fullMembers []flow.Identifier, stakingSigners flow.IdentifierList, beaconSigners flow.IdentifierList) ([]byte, []byte, error) {
```

Comment:
For me, it would be more intuitive to rename  `fullMembers` to `orderedCommittee`

Reply:
https://github.com/onflow/flow-go/pull/2204/files#diff-a1983c6bd364d16ae09eb3dcc8abd9970e72cc66df5e8901d1434f188567d5b3R233